src/Request/RequestHandler.php

```diff
@@ -197,7 +213,12 @@ public function handleRequest(string $method, string $uri, array $options, array
             $this->handleException($exception);
         }
 
-        return json_decode($response->getBody());
+        $contents = $response->getBody()->getContents();
+
+        if ($contentMethod) {
```

Comment:
I see

I don't see anything passing in `parse_str` though, can you show me where that happens?